recipe/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "trulens-core" %}
+{% set version = "1.0.7" %}
```

```suggestion
{% set version = "1.1.0" %}
```
It seems the latest that still falls within the range for trulens-feedback is 1.1.0. Is there any particular reason to not use the latest?

---

Going to update to `1.1.0`. We will be trying to keep all the `trulens` packages at around the same versions.